### ISSUE TITLE
DWIPreproc references non-existing inputs

### ISSUE THREAD

**Issue by GalKepler** [nipy/nipype#3656]
### Summary
In the _list_outputs method of mrtrix's DWIPreproc (in [nipype.interfaces.mrtrix3.preprocess](https://github.com/nipy/nipype/blob/master/nipype/interfaces/mrtrix3/preprocess.py)).
### Actual behavior
Raises an error due to referencing `export_grad_mrtrix` and `export_grad_fsl`, while the actual inputs are `export_grad_mrtrix` and `out_grad_fsl`.
### How to replicate the behavior
Simply execute DWIPreproc with any given inputs.

### Platform details:

```
{'commit_hash': '57c9932',
 'commit_source': 'repository',
 'networkx_version': '3.3',
 'nibabel_version': '5.2.1',
 'nipype_version': '1.8.6',
 'numpy_version': '1.26.4',
 'pkg_path': '/home/galkepler/Projects/yalab-devops/yalab-procedures/.venv/lib/python3.11/site-packages/nipype',
 'scipy_version': '1.14.0',
 'sys_executable': '/home/galkepler/Projects/yalab-devops/yalab-procedures/.venv/bin/python',
 'sys_platform': 'linux',
 'sys_version': '3.11.8 | packaged by conda-forge | (main, Feb 16 2024, '
                '20:53:32) [GCC 12.3.0]',
 'traits_version': '6.3.2'}
```

### Execution environment
- My python environment outside container

**Comment by GalKepler** [comment]
Found out that I've already fixed this issue before at #3596, but for some reason my fix isn't included in the PyPi package...

**Comment by effigies** [comment]
I think we need to release
